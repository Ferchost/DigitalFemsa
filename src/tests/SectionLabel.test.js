import React from 'react';
const { default: SectionLabel } = require("../components/SectionLabel");
import renderer from 'react-test-renderer';

let component;
test("Renderiza bien",()=>{
        component = renderer.create(<SectionLabel text={"Hola Test"} UpperCase={false} />).toJSON()
        expect(component).toMatchSnapshot();
  
})