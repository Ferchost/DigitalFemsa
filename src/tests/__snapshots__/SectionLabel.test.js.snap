// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renderiza bien 1`] = `
<Text
  style={
    {
      "alignSelf": "flex-start",
      "color": "#9B9898",
      "fontSize": 15,
      "fontWeight": "bold",
      "margin": 15,
      "marginLeft": "6%",
      "textAlign": "left",
    }
  }
>
  Hola Test
</Text>
`;
